lib/signs_ui_web/templates/layout/app.html.eex

```diff
@@ -10,6 +10,9 @@
     <title>MBTA Realtime Signs</title>
     <link rel="stylesheet" href="<%= static_path(@conn, "/vendor/bootstrap.min.css") %>">
     <link rel="stylesheet" href="<%= static_path(@conn, "/css/app.css") %>">
+    <%= if record_sentry?() do %>
+      <%= render "_sentry.html", assigns %>
```

Comment:
Do you need to pass the assigns? It doesn't appear they're used in the template.

Reply:
This was copied from Skate which also doesn't use them, I wasn't aware that you could render without passing assigns.

Reply:
There's a default of empty: https://hexdocs.pm/phoenix/1.5.9/Phoenix.Controller.html#render/1

Reply:
oh nice! thanks.